CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # NVIDIA Container Toolkit Changelog
 
+## v1.17.5
```

Comment:
Thanks for adding release notes / changelog entries.

Thought: in the future it would be nice if especially "fix a bug.." would link to a ticket describing the bug.

Reply:
Yes, we could get better at tracking these things. Sometimes these are internal only and I try to avoid referencing those here.